pyproject.toml

```diff
@@ -2,20 +2,21 @@
 name = 'asdf_transform_schemas'
 description = 'ASDF schemas for transforms'
 readme = 'README.md'
-requires-python = '>=3.8'
+requires-python = '>=3.9'
 license = { file = 'LICENSE' }
 authors = [{ name = 'The ASDF Developers', email = 'help@stsci.edu' }]
 classifiers = [
     'Programming Language :: Python',
     'Programming Language :: Python :: 3',
-    'Programming Language :: Python :: 3.8',
     'Programming Language :: Python :: 3.9',
     'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Programming Language :: Python :: 3.12',
     'Development Status :: 5 - Production/Stable',
 ]
 dependencies = [
-    'asdf-standard >= 1.0.1',
-    'importlib_resources >= 3; python_version<"3.9"',
+    #'asdf-standard >= 1.0.1',
+    'asdf-standard @ git+https://github.com/braingram/asdf-standard.git@split_core'
```

Comment:
To be removed prior to merge and after https://github.com/asdf-format/asdf-standard/pull/422 is merged and released.